students/csdotson/lesson08/circle.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+import math
+
+class Circle:
+    """Create a Circle class representing a simple circle"""
+
+    def __init__(self, radius):
+        if radius < 0:
+            raise ValueError("radius can't be less than 0")
+        self._radius = radius
+
+    @property
+    def radius(self):
+        return self._radius
+
+    @radius.setter
+    def radius(self, val):
+        if val < 0:
+            raise ValueError("radius can't be less than 0")
+        self._radius = val
+
+    @property
+    def diameter(self):
+        return 2 * self.radius
+
+    @diameter.setter
+    def diameter(self, val):
+        self.radius = val / 2
+
+    @classmethod
+    def from_diameter(cls, diameter):
+        return ( cls(diameter/2) )
+
+    @property
+    def area(self):
+        return (math.pi * math.pow(self.radius, 2))
+
+    def __lt__(self, other):
+        """ Define less than comparator for circles """
+        return (self.radius < other.radius)
```

Comment:
you don't need all the parens with return statement -> `return self.radius < other.radius`